404.html

```diff
@@ -3,18 +3,44 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>Meetup</title>
+        <title>FOSSASIA | Asia's Open Technology Organization for Open Source Software, Hardware, Open Design, Open Data, Open Knowledge</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="FOSSASIA Open Tech Community - Open Source, Big Data, Design Thinking and Free Knowledge Tools with Asia's Premier Open Technology Organization">
+        <meta name="keywords" content="FOSSASIA, FOSS, Free and Open Source Software, Asia">
+        <meta name="author" content="FOSSASIA">
+        <meta property="og:title" content="FOSSASIA" />
+        <meta property="og:type" content="website" />
+        <meta property="og:image" content="http://fossasia.github.io/fossasia15/img/fossasia-dark.png" />
+        <meta name="twitter:card" content="summary">
+        <meta name="twitter:site" content="@fossasia">
+        <meta name="twitter:title" content="FOSSASIA">
+        <meta name="twitter:description" content="FOSSASIA Open Technology Community - Open Source, Big Data, Design Thinking and Free Knowledge Tools with Asia's Premier Open Technology Organization">
+        <meta name="twitter:creator" content="@fossasia">
+        <meta name="twitter:image" content="http://fossasia.github.io/fossasia15/img/fossasia-dark.png">
+
+        <link rel="shortcut icon" href="fossasia.ico" type="image/x-icon" />
         <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/flexslider.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/elegant-icons.min.css" rel="stylesheet" type="text/css" media="all"/>
+        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/pe-icon-7-stroke.min.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/theme.css" rel="stylesheet" type="text/css" media="all"/>
```

Comment:
What about `<link href="css/theme.css" rel="stylesheet" type="text/css" media="all"/>` 
You deleted that too :open_mouth: